libs/platform/src/lib/components/menu-button/menu-button.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, ChangeDetectionStrategy, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { InputSize } from './../form/form-control';
+import { ButtonType } from '../button/button.component';
+import { ButtonOptions } from '../button/button.component';
+
+@Component({
+    selector: 'fdp-menu-button',
+    templateUrl: './menu-button.component.html',
+    styleUrls: ['./menu-button.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MenuButtonComponent implements OnInit {
+    /** Id for menu-button. */
+    @Input()
+    Id: string;
+
+    /** Option to make to menu-button compact. */
+    @Input()
+    size: InputSize = 'cozy';
+
```

Comment:
It's not a comment, just a  question. Why do you go with this approach of having `size` and `iscompact`? You can have an Input property called `compact` which can be boolean and set to false by default. If you need the button to be in compact mode you can set it to true. Having size property can be confusing to the user.

Reply:
we are following this in Platform as per Frank comments in other components.

Reply:
Ok. Thank you for answering my question

Reply:
> Ok. Thank you for answering my question

@InnaAtanasova  I dont really see any explaination ;-(,  @DeepakSap14 you should try to explain as `per Frank` does not mean much and does not tell a thing to anyone ;-(

Reply:
We have introduced `size: Compact | cozy` for input Widgets and it is defined in the BaseInput. The reason for this was since we know that every INPUT is going to expand 100% to its parent container so most likely we are not going to use native `size` attribute and this cozy | compact is kind of size so we used this so every component have unified naming for this. From expirience we expect it will not end only with these two compact and cozy, they might be other.  I am still thinking about some better name.. 

This was for Inputs. As for the menu button, can you define the actual width? 

For the non-input and I see there is a `width` in this case it might be confusing size, width .

Reply:
Thank you for the detailed explanation, @fkolar It makes sense. From "we are following this in Platform as per Frank comments in other components" I understood that this is an approach you agreed on for the Platform and it has been discussed with you.